src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -229,11 +234,27 @@ const CreateCacheWizard = (props) => {
       let activeButton = true;
       switch (activeStepId) {
         case 1: activeButton = configuration.start.valid; break;
-        case 2: activeButton = true; break;
+        case 2:
+          if(ConsoleServices.security().hasConsoleACL(ConsoleACL.CREATE, connectedUser)){
```

Comment:
active button is already init to true. check if the user has not the right and set it to false instead and break